content/blog/excel-to-r.md

```diff
@@ -0,0 +1,72 @@
+---
+title: "Experiments in transforming Excel into R"
+date: 2019-11-18
+---
+
+Last week was the [Health Economics in R Hackathon](https://n8thangreen.wixsite.com/hermes-hack-london), bringing together health economists, epidemiologists and software engineers.
+
+We learned that complex spreadsheet models are common in health economics, and that the users of these models feel constrained by what they can achieve with them - having constructed a [decision tree model](https://en.wikipedia.org/wiki/Decision_tree) in Excel, the analyst might want to carry out a [sensitivity analysis](https://en.wikipedia.org/wiki/Sensitivity_analysis) vary the models' parameters seeing how the outcomes vary.  Doing this requires knowledge of [Visual Basic](https://en.wikipedia.org/wiki/Visual_Basic_for_Applications) and reportedly is quite slow to run.  We wanted to create a proof-of-concept package that would take an excel model like this and convert it into an R function, where it could be more easily manipulable.
```

Comment:
The grammar seems a little bit off at 'sensitivity analysis vary the models' parameters'. Should be 'to vary..' or 'by varying..' or something? Capitalise 'excel' in the last sentence'?